enterprise/actions.py

```diff
@@ -58,3 +58,18 @@ def export_as_csv(modeladmin, request, queryset):  # pylint: disable=unused-argu
 
     export_as_csv.short_description = description
     return export_as_csv
+
+
+def get_clear_catalog_id_action(description=None):
```

Comment:
Given that the file already existed with a given name, I'd prefer to leave it as-is in this PR unless there's a compelling reason for the rename. In addition, `actions.py` is an existing Django trope.

Finally, for the function name, I think it's important to differentiate between the outer function, whose job is to *get* the *action* function, and the inner function, whose job is to actually clear the catalog ID.